Add VPC CIDR mask constraint (#989)
timescale/docs: cloud/vpc-peering.md

1.  In the `Create a VPC` dialog:
    *   Type a name for your new VPC and select the region that matches the region of the service you
        want to attach it to.
    *   Provide an IPv4 CIDR block. Make sure that your VPC CIDR block has its mask in 

>         the range between 16 and 28 and that the CIDR block you choose for your 

Reviewer: Only thing to note here for me is that it would be good for the console to protect users from entering anything out of this range.

Reviewer: https://github.com/timescale/web-cloud/issues/2282 was filed for this originally and I filed https://github.com/timescale/web-cloud/issues/2307

        Timescale Cloud VPC does not overlap with the CIDR block used by your AWS VPC
        peer. If the CIDR blocks overlap, the peering process fails. You can find the

>         CIDR block of your AWS VPC from the AWS console. In this example, we use the

Reviewer 2: ```suggestion
        CIDR block of your AWS VPC from the AWS console. This example uses the
```
